interpreter/k8s/src/main/scala/io/buoyant/transformer/k8s/DaemonSetTransformerInitializer.scala

```diff
@@ -41,8 +41,8 @@ case class DaemonSetTransformerConfig(
   private[this] val netmask = InetAddress.getByName("255.255.255.0")
 
   @JsonIgnore
-  override def mk(): NameTreeTransformer = {
-    val client = mkClient(Params.empty).configured(Label("daemonsetTransformer"))
+  override def mk(params: Params): NameTreeTransformer = {
+    val client = mkClient(params).configured(param.Label("clnt"))
```

Comment:
My gut feeling is that the conflict on "client" isn't really that onerous -- on one hand we're scoped under rt/_/... on the other, we're scoped under namer/...

I don't feel that strongly, though